cmd/agent/main.go

```diff
@@ -78,11 +80,20 @@ func main() {
 
 	mw := io.MultiWriter(os.Stdout, logFile)
 	log.SetOutput(mw)
+	// TODO make this log level configurable
+	log.SetLevel(log.DebugLevel)
 
 	nimbessConfig := agent.InitConfig(*confFile)
 	driver := selectDriver(nimbessConfig)
+	// Create Pipeline map of per port pipelines
+	nimbessPipelineMap := make(map[string]*agent.NimbessPipeline)
+	metaPipelineMap := make(map[string]*agent.NimbessPipeline)
+	// Generate ID for this Agent
+	// TODO(trozet) Make this ID stored in etcd and pull at agent start
```

Comment:
What is the uuid for? Identify the single agent? So we would have one per node? And then do we expect one of the stargazer controllers to write it? Seems easiest if Stargazer writes it since it knows when etcd is ready. We should still work on ordering architecture if needed.

Reply:
yeah UUID is to identify each agent. K8S uses the uuid on the host, I will probably change to use the same thing. There will be one per node. The node UUID will be used to create a path in etcd to store all configuration for this node. Each agent can look at its own as well as other nodes config. Stargazer should not write to a node; it should write to some nimbess reflective path that the nodes watch for. The node paths are for nodes to write to.